### PR TITLE
New fields, abstracts, AHU/MAU/CH canonical types

### PR DESCRIPTION
**Pull request by evgeniya-engel**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 1]
```diff
@@ -4159,6 +4159,91 @@ AHU_CHWSC_ECON_FDPM_HWSC_SFSS_SFVSC_SSPC_SDHC_RTM:
...
+  - SARC
+  - SSPC
+  - RD
+  - OAFMC
+  opt_uses:
+  - smoke_alarm
+
+AHU_SFSS_SFVSC_EFSS_EFVSC_CHWSC_HWSC_ECON_EDBPC_SARC_SSPC_OAFMC:
+  guid: "36c1740d-1eaa-4bae-8779-269cdfc0c5a0"
+  description: "Multi-zone cooling and heatinfg AHU with supply air reset control and supply static pressure control."
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:4231]: Please fix typo in description

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 2]
```diff
@@ -4159,6 +4159,91 @@ AHU_CHWSC_ECON_FDPM_HWSC_SFSS_SFVSC_SSPC_SDHC_RTM:
...
+  - SARC
+  - SSPC
+  - RD
+  - OAFMC
+  opt_uses:
+  - smoke_alarm
+
+AHU_SFSS2X_SFVSC2X_RFSS2X_RFVSC2X_CHWSC_HWSC_ECON_EDBPC_SARC_SSPC_RD_OAFMC:
+  guid: "5a3034e7-0a17-416c-b974-55072d52d3e4"
+  description: "Cooling and heating AHU with 2 supply and 2 return fans, supply air reset control,
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:4209]: nit: please remove carriage return in description text block (description text should just be on a single line)

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 3]
```diff
@@ -4159,6 +4159,91 @@ AHU_CHWSC_ECON_FDPM_HWSC_SFSS_SFVSC_SSPC_SDHC_RTM:
...
+  - SSPC
+  - RD
+  - OAFMC
+  - EFM
+  opt_uses:
+  - smoke_alarm
+
+AHU_SFSS4X_SFVSC4X_RFSS3X_RFVSC3X_CHWSC_HWSC_ECON_EDBPC_SARC_SSPC_RD_OAFMC:
+  guid: "dca00b39-6816-4d59-b3b6-2b3a429edf8b"
+  description: "Cooling and heating AHU with 4 supply and 3 return fans, supply air reset control,
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:4187]: nit: please remove carriage return in description text block (description text should just be on a single line)

File: ontology/yaml/resources/HVAC/entity_types/CH.yaml [hunk 4]
```diff
@@ -609,6 +609,38 @@ CH_SS_DX3SCHWC_CWSISOVM_CWRISOVM_CHWSWISOVM_CHWDPM_CWDT_CHWDT_RCKTM:
...
+  - SS
+  - DX4SWC
+  - RWTM
+  - REFSM2X
+  - REFPM2X
+  - CPC
+
+CH_SS_DX2SWC_CHDXVSC2X_RWTM_MWDPC_RWISOVM_REFSM2X_REFPM2X_CPC_INVIPWM2X_INVOPWM2X:
+  guid: "cf101310-f825-4637-9948-f036ecbb4210"
+  description: "Two stage air cooled chiller with dual compressor variable speed control, return water isolation valve, minimum differential pressure control,
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/CH.yaml:627]: nit: please remove carriage return in description text block (description text should just be on a single line)

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml [hunk 5]
```diff
@@ -712,6 +712,23 @@ MAU_SFSS2X_SFVSC2X_CHWSC_HWSC_SSPC_VOADM_FDPM_ZSPM_OAFM:
...
+  implements:
+  - MAU
+  - DFSS
+  - DFVSC
+  - DSP
+  - CHWDC
+  - HWDC
+  - OADM
+  - ZSPM
+  - OA
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/MAU.yaml:728]: Seems odd to have `OA` on this equipment, typically we only use `OA` to model stand-alone weather stations

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 6]
```diff
@@ -798,6 +801,9 @@ literals:
 - chilled_water_valve_command:
   - OPEN
   - CLOSED
+- drain_water_valve_command:
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:804]: I don't see this field used in any of the ABSTRACT types

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/CH.yaml (hunk 4 above)
Comment (evgeniya-engel) [reply ontology/yaml/resources/HVAC/entity_types/CH.yaml]: fixed

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 3 above)
Comment (evgeniya-engel) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml]: fixed

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 2 above)
Comment (evgeniya-engel) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml]: fixed

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 1 above)
Comment (evgeniya-engel) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml]: fixed

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**